docs/content/docs/data-structures/overview.md

```diff
@@ -110,3 +110,8 @@ Typical scenarios require the connected clients to "agree" on some course of act
 
 - Import data from an external source. (Multiple clients doing this could lead to duplicate data.)
 - Upgrade a data schema. (All clients agree to simultaneously make the change.)
+
+<!-- Links -->
+[SharedMap]: https://fluidframework.com/docs/apis/map/sharedmap/
```

Comment:
I believe this is preferred: `{{< relref "map.md" >}}`. That makes the link work in a wider variety of environments (e.g. running website locally)

Reply:
I was wondering about that. So writing it out fully like this refers to the packages API docs (https://fluidframework.com/docs/apis/map/sharedmap/) whereas `{{< relref "map.md" >}}` opens up the text explainer (https://fluidframework.com/docs/data-structures/map/). In this case I believe we want the links to the API docs instead of the text explainer docs, especially as the text explainers aren't all complete yet (e.g. https://fluidframework.com/docs/data-structures/counter/, https://fluidframework.com/docs/data-structures/string/).

But happy to do the relative links if you think that's where this page should link to!

Reply:
Oh--I missed that you were linking to API docs. It should still be possible to link there using the same `relref` syntax, you can find docs [here](https://gohugo.io/functions/relref/). Without actually trying it myself (don't have a repo checked out that builds right now), I think this might work? `{{< relref "/docs/apis/map/sharedmap" >}}`

Reply:
Oh cool, let me see if that works and if so I'll change em out!

Reply:
talked to Kajari offline on this, seems like there are issues linking to API docs this way. Perhaps it's related to build order of the website w.r.t API docs (something along the lines of them not being generated "soon enough"). Going to check in the always-pointing-to-FF-website links for now and fix them to use relref later.